.github/workflows/package-windows.yml

```diff
@@ -108,14 +108,21 @@ jobs:
         run: python3 freeze_program.py ${{ github.workspace }} --wheel-it-dir ${{ github.workspace }}/dist
         working-directory: ${{ runner.temp }}
 
-      - name: Build NSIS setup
-        run: makensis.exe installer.nsi
-        working-directory: ${{ runner.temp }}
-
       - name: Debug build directory
         run: Get-ChildItem -Recurse -Path ${{ runner.temp }}\build
 
+      # Cannot do the NSIS installer part in CI given it requires to sign `parsec.exe`
+
+      - name: Prepare artifact
+        run: |
+          Copy-Item -Path build\manifest.ini -Destination dist
+          Copy-Item -Path build\install_files.nsh -Destination dist
+          Copy-Item -Path build\uninstall_files.nsh -Destination dist
+          Copy-Item -Recurse -Path build\parsec-* -Destination dist
+          Copy-Item -Path build\winfsp-* -Destination dist
+        working-directory: ${{ runner.temp }}
+
       - uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # pin v3.1.0
         with:
           name: ${{ runner.os }}-${{ runner.arch }}-installer
```

Comment:
Maybe rename the generated artifact ?

Reply:
what for ?

Reply:
we need them with their original name so that we can locally extract the artifact archive then run `make_installer.py`

Reply:
the artifact now don't contains the installer but the files to generate the installer
as you renamed the job to `[...]-installer-content`

Reply:
Ho you're right ! I missed this field ;-)